packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -286,9 +286,7 @@ const browserStatus = computed(() => {
 const testingTypeIcon = computed(() => props.gql.currentTestingType === 'component' ? TestingTypeComponentIcon : TestingTypeE2EIcon)
 
 onMounted(() => {
-  if (props.wasBrowserSetInCli && !browserStatus.value.open) {
-    emit('launch')
-  }
+  props.launchIfBrowserSetInCli()
```

Comment:
I passed as a prop and used `onMounted` because when the component mounts, the data inside the function will be available. I am still struggling with making sure it only launches once when mounted. Currently when you navigate back and select the same testing type... it still relaunches based on the conditions in the function. From what I can tell in some of the actions in data context, this should be normal behavior to relauch  IF the testing type is changed otherwise this shouldn't occur. 

I have tried a few different things on the client to get it to work, but they aren't viable solutions and always seem to have an edge case that can't be handle on the client based on the information currently available. Any suggestions for this?